### PR TITLE
(SIMP-5081) Add missing params

### PR DESCRIPTION
**Pull request by jeefberkey**
- Add package_ensure parameters to packages managed by this module
  - $scap_ensure - openscap-utils
  - $ssg_ensure - scap-security-guide
- Add missing $scap_tailoring_file parameter to openscap::schedule

SIMP-5081 #close

### CODE REVIEW

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-openscap/pipelines/31717778

File: manifests/schedule.pp [hunk 1]
```diff
@@ -52,6 +52,7 @@
   Stdlib::Absolutepath             $oscap_path             = pick(fact('oscap.path'), '/bin/oscap'),
   Stdlib::Absolutepath             $ssg_base_dir           = '/usr/share/xml/scap/ssg/content',
   Boolean                          $fetch_remote_resources = false,
+  Optional[Stdlib::Absolutepath]   $scap_tailoring_file    = undef,
```
Comment (lnemsick-simp) [inline manifests/schedule.pp]: Please update the parameter documentation.

**Comment by jeefberkey** [comment]
Acceptance should pass now as well. https://gitlab.com/simp/pupmod-simp-openscap/pipelines/31952694

File: manifests/init.pp [hunk 2]
```diff
@@ -4,14 +4,23 @@
 #
 # @author https://github.com/simp/pupmod-simp-openscap/graphs/contributors
 #
-class openscap(
-  Boolean $enable_schedule = false
-){
+class openscap (
```
Comment (lnemsick-simp) [inline manifests/init.pp]: please add parameter documentation

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-openscap/pipelines/32146255

**Pull request merged**